### PR TITLE
Enable Autofill for existing DDG users feature flag and pixels

### PR DESCRIPTION
**Pull request by graeme**
Task/Issue URL: https://app.asana.com/0/0/1207868278261135/f

**Description**:
- Adds a feature flag for gradual roll-out of the enabling of autofill for existing DDG users who hadn't already used it.
- Adds Pixels to monitor the performance of the changes made in https://github.com/duckduckgo/iOS/pull/3170
- For Pixel definitions https://app.asana.com/0/414235014887631/1207868278261135/f

**Steps to test this PR**:
**For the Pixels:**
1. Install a clean build
2. Go to fill.dev/form/login-simple and submit some details
3. Check that you see the `autofill.logins.save.login.inline.onboarding.displayed` pixel in the debugger
4. Dismiss the prompt by pressing the x or swiping down
5. Check that you see the `autofill.logins.save.login.inline.onboarding.dismissed` pixel in the debugger
6. Repeat step 2
7. Tap Save Password
8. Check that you see the `autofill.logins.save.login.inline.onboarding.confirmed` pixel in the debugger
9. Repeat steps 1 and 2
10. Tap the "Never for this website" button
11. Check that you see the `autofill.logins.save.login.onboarding.exclude.site.confirmed` pixel in the debugger
12. Go to any other website with a login and log in
13. Tap either dismiss or "Never for this website"
14. Check that you see the `autofill.logins.save.disable.snackbar.shown` pixel in the debugger
15. Tap the "Open Settings" button
16. Check that you see the `autofill.logins.save.disable.snackbar.open.settings` pixel in the debugger and...
17. Check that you see the `m.autofill.management.opened ["source": "save_login_disable_prompt"]` pixel in the debugger
18. Disable autofill
19. Check that you see the `m.autofill.logins.settings.disabled ["source": "save_login_disable_prompt"]` pixel in the debugger

**Definition of Done (Internal Only)**:

* [ ] Does this PR satisfy our [Definition of Done](https://app.asana.com/0/1202500774821704/1207634633537039/f)?

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 15
* [ ] iOS 16
* [ ] iOS 17

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Comment by amddg44** [comment]
@graeme since this is a stacked PR I believe, can the branch this is merging into be changed from `main`? it’s hard to see the code changes from just this PR, thanks

**Comment by graeme** [comment]
@amddg44 Ach sorry, forgot to do that. It's fixed now!

File: DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -138,7 +138,7 @@
     {
       "identity" : "swift-argument-parser",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/apple/swift-argument-parser",
+      "location" : "https://github.com/apple/swift-argument-parser.git",
```
Comment (amddg44) [inline DuckDuckGo.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:141]: Just FYI a `.git` got added back in here

**Review by amddg44 — APPROVED** [review]
LGTM! 🚀

**Pull request merged**